src/structlog/processors.py

```diff
@@ -41,7 +41,13 @@
 )
 from ._log_levels import _NAME_TO_LEVEL, add_log_level
 from ._utils import get_processname
-from .types import EventDict, ExcInfo, WrappedLogger
+from .tracebacks import DictFormatter as DictExceptionFormatter
```

Comment:
Why the `as`?

Reply:
To make it clear for users of that package, that `DictFormatter` is not a log-formatter, but an exception formatter.

`DictFormatter` is a good name if you take its module name into account, but with `from` imports, this information is lost.

But if you want the `as ...` to be removed, I will remove it. :)

Reply:
I would suggest to call the formatter DictExceptionFormatter. I know that structlog is a bit wild on not doing stutter imports (we've got like 2094 `BoundLogger`s), but in hindsight it was a mistake I think.